src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,18 @@
+const { ApiError } = require('../utils/api.error');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+
+    return;
+  }
+
+  res.status(500).send({
+    message: 'Server error',
+  });
```

Comment:
Consider adding logging for unexpected errors to help with debugging and monitoring. This can be done using `console.error(error)` or a more sophisticated logging solution.